components/event-bus/cmd/mesh-namespaces-migration/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM curlimages/curl:7.66.0
```

Comment:
can we update the curl image, as per this: CxServer\SP\SAP\OpenSource

Reply:
@anishj0shi could you please indicate the full image name ?

Reply:
my bad, i thought the link was updated.. https://nvd.nist.gov/vuln/detail/CVE-2019-15601

Reply:
That's the CVE, what about the image name?

Reply:
I meant: what image should we use?

Reply:
image version: 7.68

Reply:
[done](https://github.com/kyma-project/kyma/compare/9f3745fbd22aa842cbb6307d4b7f138983bab958..5d7bc22aa931c741d78725979e7a7575c63413d2)